modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/dataset_display/utilities/index.js

```diff
@@ -88,15 +105,20 @@ export function loadData(
 ) {
 	const authString = `p_auth=${window.Liferay.authToken}`;
 	const currentUrlString = `&currentUrl=${encodeURIComponent(currentUrl)}`;
-	const pagination = `&pageSize=${delta}&page=${page}`;
+	const paginationString = `&pageSize=${delta}&page=${page}`;
 	const searchParamString = searchParam ? `&search=${searchParam}` : '';
 	const sortingString = sorting.length
-		? `&orderBy=${JSON.stringify(sorting)}`
+		? `&sort=${sorting
+				.map((item) => `${item.key}:${item.direction}`)
+				.join(',')}`
 		: ``;
+	const filtersString = filters.length
+		? `&filter=${createOdataFilter(filters)}`
+		: '';
 
 	const url = `${apiUrl}${
 		apiUrl.indexOf('?') > -1 ? '&' : '?'
-	}${authString}${currentUrlString}${pagination}${sortingString}${searchParamString}`;
+	}${authString}${currentUrlString}${paginationString}${sortingString}${filtersString}${searchParamString}`;
```

Comment:
Same story here, but it would probably benefit even more from the `URL`/`searchParams` treatment.

Reply:
This looks a bit like [Brainfuck](https://copy.sh/brainfuck/text.html). Tell me if you can see the difference

```
+++++++++.++[--->++++<]>-.++++++++++++.-.+++++.+.[---->+<]>+++.[->+++<]>+.--[--->+<]>.-.------------.>-[--->+<]>--.[--->+<]>+++.--.---------.+++++.-------.-[--->+<]>--.-[->++<]>-.+[-->+<]>+.[->+++<]>.[++>-----<]>.[----->++<]>-.++.--[--->+<]>.-.------------.+[->+++<]>++.-[----->+++<]>.+++++[->+++<]>.----.+[->+++<]>+.+++++.----------.+++++++++++.++++++++.---[++>---<]>.-[--->+++++<]>+.++[--->++++<]>+.---.-.+++++++++++++.+++[->+++<]>++.+[--->+<]>+++.-----[++>---<]>.++[->++<]>+.--[--->+<]>.-.------------.>-[--->+<]>-.-[--->+<]>--.----.------.+++++++++.[--->+<]>+++.+++[---->+++<]>.-[->++++++<]>+.>++++++++++.-.++[--->++++<]>-.++++++++++++.-.+++++.+.[---->+<]>+++.+[->+++<]>.-[--->+<]>-.---..-------------.+++++++++.++++++.>-[--->+<]>.+[--->+<]>.------.>-[--->+<]>--.[--->+<]>+++.--.---------.+++++.-------.-[--->+<]>--.-[->++<]>-.+[-->+<]>+.[->+++<]>.++[->+++<]>.[--->+<]>+.-[--->+<]>-.---..-------------.+++++++++.++++++.>-[--->+<]>.+[--->+<]>.------.--[->+++<]>-.-[----->+++<]>.+++++[->+++<]>.[->+++++<]>--.+++++++++.-----------.++++++++++++.-----------.+.>-[--->+<]>.---.---------.------.+[--->+<]>+++.--.+++.-.-.---------.+++++++++.++++++.+[--->+<]>+.+[--->+<]>.-[--->+<]>-.---..-------------.+++++++++.++++++.>-[--->+<]>.+[--->+<]>.------.[--->+<]>+++++.[->+++<]>++.+++[---->+++<]>.-[->++++++<]>+.>++++++++++.-.++[--->++++<]>-.++++++++++++.-.+++++.+.[---->+<]>+++.[-->+++++++<]>.[----->++<]>+.++++++.++.+++++.-------------.--[--->+<]>-.-----------.++++++.-.+++[->+++<]>.[--->+<]>+++.--.---------.+++++.-------.-[--->+<]>--.-[->++<]>-.+[-->+<]>+.[->+++<]>.++[->+++<]>.-[->+++<]>+.[----->++<]>+.++++++.--.>-[--->+<]>--.+[---->+++++<]>.-[--->+<]>++.---[->+++<]>.-[++++>-----<]>.-[----->+++<]>.+++++[->+++<]>.-[----->++<]>.+.+++++++.++++++++.+[->+++<]>++.>--[-->+++<]>.--[--->+<]>---.-[->+++<]>+.[----->++<]>+.++++++.--.-[++++>-----<]>.-[----->+++<]>.+++++[->+++<]>.[->+++<]>-.[----->++<]>+.++++++.--.>--[-->+++<]>.+++[---->+++<]>.-[->++++++<]>+.>++++++++++.-.++[--->++++<]>-.++++++++++++.-.+++++.+.[---->+<]>+++.---[->++++<]>-.++++[->+++<]>.----.--[--->+<]>---.++++[->+++<]>+.+++++.[---->+++<]>++.[------->++<]>+.--[--->+<]>---.+++[->+++<]>++.++++++++++++.>-[--->+<]>--.[--->+<]>+++.--.---------.+++++.-------.-[--->+<]>--.-[->++<]>-.+[-->+<]>+.---[->++++<]>-.++++[->+++<]>.----.--[--->+<]>---.++++[->+++<]>+.+++++.[---->+++<]>++.[------->++<]>+.--[--->+<]>---.+++[->+++<]>++.++++++++++++.[->+++++<]>-.[->++<]>-.+[-->+<]>.[->+++<]>.++[->+++<]>.[->+++<]>+.++++[->+++<]>.----.--[--->+<]>---.++++[->+++<]>+.+++++.+[->+++<]>++.-[----->+++<]>.+++++[->+++<]>.--------.++++[->+++<]>.----.--[--->+<]>---.++++[->+++<]>+.+++++.[---->+++<]>++.[------->++<]>+.--[--->+<]>---.+++[->+++<]>++.++++++++++++.>--[-->+++<]>.+++[---->+++<]>.[->+++<]>.---[->++<]>.[-->+<]>+++.+++++++..[----->+<]>.>++++++++++.-.++[--->++++<]>-.++++++++++++.-.+++++.+.[---->+<]>+++.---[->++++<]>-.----.+++.++.-----------.+++++.-------.>-[--->+<]>--.[--->+<]>+++.--.---------.+++++.-------.-[--->+<]>--.-[->++<]>-.+[-->+<]>+.---[->++++<]>-.----.+++.++.-----------.+++++.-------.--[->+++<]>-.--[-->+++++<]>--.-------.+++++++++.-------.+++++++++++++.------------.>++++++++++.-..[->+++++++<]>.+[-->+<]>.[->+++<]>.++[->+++<]>.[->+++<]>+.----.+++.++.[-->+<]>+++.-[----->+++<]>.+++++[->+++<]>.--------.----.+++.++.-----------.+++++.-------.>++++++++++.-....[->+++++<]>+.--[-->+++++<]>-.------------.-[++>-----<]>.[++>---<]>..-----[->+++<]>.+++++++++++.+++[->+++<]>.++++++++.+++[++>---<]>+.---------.-[->++<]>-.+.[-->+<]>+.[->+++<]>.+[->+++<]>+.+++++[->+++<]>.[->+++++<]>++.+++++++++++.+++[->+++<]>.++++++++.+[----->++<]>++.[---------->+<]>.------.[--->+<]>++.++++.-[++>---<]>.[->+++++<]>++.+++++[->+++<]>.[->+++++<]>++.+++++++++++.+++[->+++<]>.++++++++.+[----->++<]>++.--[--->+<]>.+++++.+++++++++.-------------.--.-[--->+<]>--.-----------.++++++.-.[--->+<]>+++.+++[---->+++<]>.+++[->+++<]>.>++++++++++.-....[->+++++<]>+.+++++++[->++<]>.+++++.------.+++++.++[++>---<]>.-.+++++.-----.++.[->+++<]>++.+++[---->+++<]>.>++++++++++.-..+[->++++++<]>--.[-->+<]>+++.[->+++<]>..-[->++++++<]>+.>++++++++++.-.++[--->++++<]>-.++++++++++++.-.+++++.+.[---->+<]>+++.++[->+++<]>.+++.+++.++++++++.+++[->+++<]>.+++++++++++++.+.--[->+++<]>.[--->+<]>+++.--.---------.+++++.-------.-[--->+<]>--.-[->++<]>-.+[-->+<]>+.++[->+++<]>.+++.+++.++++++++.+++[->+++<]>.+++++++++++++.+.+[++>---<]>.--[-->+++++<]>--.-------.+++++++++.-------.+++++++++++++.------------.>++++++++++.-..[->+++++++<]>.+[-->+<]>.[->+++<]>.++[->+++<]>.----[->+++<]>.+++.+++.++++++++.+++[->+++<]>.+++++++++++++.[-->+<]>++++.-[----->+++<]>.+++++[->+++<]>.-[->+++++<]>+.-[--->+<]>----.-------------.----.--[--->+<]>-.+++[->+++<]>.-[----->++++<]>-.+[---->+++++<]>.---.--[--->+<]>-.+[->+++<]>++.[->++++++<]>.[-->+++<]>.+++.++++++++.+++[->+++<]>.+++++++++++++.[--->+<]>++.+[--->+<]>+++.+++.+++.++++++++.+++[->+++<]>.+++++++++++++.+.-[--->+<]>+++.[->+++<]>++.+++[---->+++<]>.>++++++++++.-..+[->++++++<]>--.[-->+<]>+++.+++++++..[----->+<]>.>++++++++++..-.++[--->++++<]>-.++++++++++++.-.+++++.+.[---->+<]>+++.---[->++++<]>+.---.------.[++>---<]>--.-[->++<]>-.+[-->+<]>+.[->+++<]>.+[->+++<]>+.+++++[->+++<]>.-[->+++++<]>-.-[++>-----<]>.-------.>-[--->+<]>.+[--->+<]>.------.[----->+<]>+.-[--->+++++<]>.+++++[->+++<]>.>++++++++++.-..-[--->++++<]>+.-[++>-----<]>.-------.>-[--->+<]>.+[--->+<]>.------.[-->+<]>--------.+[--->+<]>++++.+++++.----------.+.[--->+<]>+.[--->++<]>-.>--[----->+<]>.---[->+++<]>-.-.+++[-->+++<]>.[--->++<]>---.++.---------.-[->++<]>.[-->+<]>+.--[-->+++<]>.++++.-[--->++<]>.[->++<]>-.+[-->+<]>.+++++++.-.+.-------.---[->++<]>.[-->+<]>+++.+++++++.+++[-->+++<]>.[--->++<]>---.>++++++++++.-.>--[-->+++<]>.-[--->+++++<]>.+++++[->+++<]>.-[->+++++<]>-.--[--->+<]>.-.------------.>-[--->+<]>--.[--->+<]>+++.--.---------.+++++.-------.>--[-->+++<]>.-[--->+++++<]>.+++++[->+++<]>.-[->+++++<]>+.-[--->+<]>-.---..-------------.+++++++++.++++++.>-[--->+<]>.+[--->+<]>.------.>-[--->+<]>--.[--->+<]>+++.--.---------.+++++.-------.>--[-->+++<]>.-[--->+++++<]>.+++++[->+++<]>.[->+++<]>-.[----->++<]>+.++++++.++.+++++.-------------.--[--->+<]>-.-----------.++++++.-.+++[->+++<]>.[--->+<]>+++.--.---------.+++++.-------.>--[-->+++<]>.-[--->+++++<]>.+++++[->+++<]>.--------.----.+++.++.-----------.+++++.-------.>-[--->+<]>--.[--->+<]>+++.--.---------.+++++.-------.>--[-->+++<]>.-[--->+++++<]>.+++++[->+++<]>.[->+++++<]>-.+++.+++.++++++++.+++[->+++<]>.+++++++++++++.+.--[->+++<]>.[--->+<]>+++.--.---------.+++++.-------.>--[-->+++<]>.-[--->+++++<]>.+++++[->+++<]>.--------.++++[->+++<]>.----.--[--->+<]>---.++++[->+++<]>+.+++++.[---->+++<]>++.[------->++<]>+.--[--->+<]>---.+++[->+++<]>++.++++++++++++.>-[--->+<]>--.[--->+<]>+++.--.---------.+++++.-------.>--[-->+++<]>.+++[---->+++<]>.-[->++++++<]>+.>++++++++++.
```

😂

Reply:
![unimpressed](https://user-images.githubusercontent.com/7074/86568380-13a2d380-bf6d-11ea-969b-8b4e313fdd7c.gif)